### PR TITLE
LPS-85419 Web content comments subscriber recevices an email after update of the web content

### PR DESCRIPTION
**Pull request by adolfopa**
@ealonso @brandizzi This fixes some Journal and Calendar upgrades, so
you should be aware of the changes.

### CODE REVIEW

**Comment by brandizzi** [comment]
@inacionery @rafaprax this PR will update Calendar.

**Comment by brianchandotcom** [comment]
Merged. Thank you.
View total diff: https://github.com/brianchandotcom/liferay-portal/compare/2f267620ec...705c8f317d

**Comment by brianchandotcom** [comment]
@adolfopa minor change in 705c8f3

@adolfopa see comment by @inacionery on bad version number

File: modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/upgrade/CalendarServiceUpgrade.java [hunk 1]
```diff
@@ -82,66 +85,63 @@ public void register(Registry registry) {
...
+				UpgradeDiscussionSubscriptionClassName.DeletionMode.ADD_NEW));
+
+		registry.register(
+			"3.0.1", "4.0.0",
+			new UpgradeDiscussionSubscriptionClassName(
+				_subscriptionLocalService, CalendarBooking.class.getName(),
+				UpgradeDiscussionSubscriptionClassName.DeletionMode.ADD_NEW));
+
+		registry.register(
+			"3.0.1", "4.0.0",
```
[the remark this answers is not in the document]
Comment (robertoDiaz) [reply modules/apps/calendar/calendar-service/src/main/java/com/liferay/calendar/internal/upgrade/CalendarServiceUpgrade.java]: Hi, yes. 

I think it was a rebase problem. I'll push a fix today

Thank you for the heads up :)

**Pull request closed without merging**